test/tests/fuel-cycle/tests

```diff
@@ -0,0 +1,10 @@
+[Tests]
+  issues = '#82'
+  design = 'ParsedODEKernel.md ODETimeDerivative.md'
+  [Abdou2021]
+    type = Exodiff
+    input = 'fuel_cycle.i'
+    exodiff = 'fuel_cycle_out.e'
+    requirement = 'The system shall faithfully reproduce a consistent solution to an ODE system of equations modeling the tritium fuel cycle.'
```

Comment:
```suggestion
    requirement = 'The system shall reproduce a consistent solution to an ODE system of equations modeling the tritium fuel cycle.'
```
@cticenhour is `faithfully` needed for SQA/standard practice? It seems like an unnecessary word to me here, but you're the expert.

Reply:
It is not needed - the requirement itself should imply `faithfully`, else we aren't doing our job 😄